meta/parse.pl

```diff
@@ -1569,6 +1571,8 @@ sub ProcessDefaultValueType
 
     return "SAI_DEFAULT_VALUE_TYPE_CONST" if $default eq "\"\"";
 
+    return "SAI_DEFAULT_VALUE_TYPE_CONST" if $default =~ /^0\,0$/;
```

Comment:
not need for \\, since comma is not regex special

Reply:
Corrected